tests/Models/User.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Tests\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use NetworkRailBusinessSystems\UserLogin\Traits\ExistingUser;
+use NetworkRailBusinessSystems\UserLogin\Traits\GetExistingUser;
+
+class User extends Authenticatable implements ExistingUser
+{
+    use GetExistingUser;
```

Comment:
Shouldn't this have its own implementation for DB, rather than LDAP?

Providing a trait for DB and LDAP would make sense.

Reply:
updated the trait which handle both ldap or local db model, and return uniqueIdentifier depeding on configuratuion options.